presto-docs/src/main/sphinx/clients/presto-console.rst

```diff
@@ -35,3 +35,23 @@ After starting Presto, you can access the web UI at the default port
 
 .. figure:: ../images/presto_console.png
    :align: center
+
+Plan Viewer
+=======================
+
+Plan Viewer is a tool to generate the graph of a query plan on the Presto Console.
+You can access it using this path: ``/ui/dev/index.html`` along with your server IP
+and port number. For example: ``http://localhost:8080/ui/dev/index.html``. When a
```

Comment:
```suggestion
and port number. For example: 

.. code-block:: none
    http://localhost:8080/ui/dev/index.html

```
Suggest changing the formatting of the URI to a code block, like the one earlier on the page, so that a user can copy to Clipboard easier.

Reply:
fixed